Sources/ViewModel/Mediatable.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftUI
+
+public protocol Mediatable: ObservableObject {
```

Comment:
Fancy name... I don't hate it, but I don't love it. It definately had a service vibe to me. 
`ViewService`
`ViewServicable`
`ViewModelProvider`

Reply:
It is from the Mediator pattern, https://en.wikipedia.org/wiki/Mediator_pattern.

I was thinking about `ViewModeling` or `ViewManaging`. But naming it `Mediatable` names it according to what a ViewModel is to a View. 

Service doesn't feel right as it is the ViewModel functionality itself and not some Service of a ViewModel.